pyproject.toml

```diff
@@ -1,13 +1,12 @@
 [tool.poetry]
-name = "sceptre-resolver-template"
+name = "jinja-unquote-resolvers-filter"
 version = "0.0.1"
-packages = [{ include = "resolver" }]
```

Comment:
you need this and since you change the folder name to `filter` it should be `packages = [{ include = "filter" }]`

Reply:
I renamed the folder back to the default insteaad.